.gitignore

```diff
@@ -1 +1,5 @@
 data/leaflet.sqlite
```

Comment:
If this is old site asset, can this entry be removed?

Reply:
Good call 👍